src/sponge.rs

```diff
@@ -0,0 +1,7 @@
+use crate::{JubJubAffine, JubJubExtended, JubJubScalar};
+use poseidon252::sponge::sponge::sponge_hash;
+
+pub fn hash(p: &JubJubExtended) -> JubJubScalar {
```

Comment:
I think it's important to document here that we need to hash the Affine representation, and why is it done like this. Since on this way on the future no one else will have the same issue you had @ZER0

Reply:
You mean the line below, not this one, where I had to convert the Extended Point to Affine Point? I'll do that.

Reply:
Exactly. I think either inside or as a fn comment. This needs to be commented.